src/utils/DragAndDrop.js

```diff
@@ -332,13 +335,17 @@ define(function (require, exports, module) {
          * or not a mime type we care about, reject it.
          */
         function rejectImport(item) {
-            if (item.size > byteLimit) {
+            var ext = Path.extname(item.name).replace(/^\./, "").toLowerCase();
```

Comment:
I think that's a good idea, actually.  @MargarytaChepiga could you add an `isArchive()` method to this file https://github.com/mozilla/brackets/blob/master/src/filesystem/impls/filer/lib/content.js.  It could do basically the same thing you're doing here.  Also, you could then change all of these long checks for zip/tar to just be `Content.isArchive()`.  Make sense?